test/BuiltIns/rlexe.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
+<regress-exe>
+  <test>
+    <default>
+      <files>ChakraLib.js</files>
+      <compile-flags>-args summary -endargs</compile-flags>
+      <tags>BuiltIn LdChakraLib</tags>
```

Comment:
><tags>BuiltIn LdChakraLib</tags> [](start = 6, length = 32)

Tags need to be a comma-separated list of tokens with no spaces (not even around the comma, as that causes anything after the first tag to be ignored by rl.exe).

Tags are used to select which tests to run or exclude from a given run of unit tests. If you intend to add either of these tags to be excluded from certain test runs, you should do that, for example a test run of a build where the binary was built with the BuiltIns explicitly turned off, you would want to run your test suite with `-nottags BuiltIn`. We did this with Intl because the Intl object is not necessarily enabled (for example it is not currently enabled on xplat). All the Intl tests are tagged as Intl.

It makes sense to have a config which disables tests tagged BuiltIn, but I'm not sure if it makes sense to disable tests when the LdChakraLib bytecode is not available. Will this ever be the case?